data_structures/src/superblock.rs

```diff
@@ -438,6 +438,11 @@ impl SuperBlockState {
         self.votes_mempool.get_received_votes()
     }
 
+    /// Returns the current committee
+    pub fn get_committee(&self) -> HashSet<PublicKeyHash> {
+        self.signing_committee.clone()
```

Comment:
You should be able to remove this clone and use a reference instead

```suggestion
    pub fn get_committee(&self) -> &HashSet<PublicKeyHash> {
        self.signing_committee
```